build/upgrade/Dockerfile

```diff
@@ -45,6 +46,7 @@ else \
 		"pgaudit${PG_MAJOR//.}*" \
 		"pgnodemx${PG_MAJOR//.}" \
 		unzip \
+		krb5-workstation \
```

Comment:
⛏️ Does it matter what order the packages are listed? If not, I'd prefer they stay alphabetical.

Reply:
Package has been properly alphabetized in this Dockerfile.